src/components/comments/CommentSection.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import Question from "./Question.js";
+import Answer from "./Answer.js";
+import "./comments.css";
+export default function CommentSection(props) {
+  let followupSection = [];
+
+  /*The function below takes each element within the followups prop array and will render its 
+    corresponding question/answer if it exists */
+  props.followups.forEach((followup) => {
```

Comment:
Right now, the labContent is set up so that each followup object can have a question AND answer component, but separating the two might make it a lot easier and would be virtually the same thing. It would also allow mapping, so we'll prob just restructure labContent again

Reply:
Alright, refactored it so that it's one-to-one and can be mapped